src/views/SystemIntake/ContractDetails/index.tsx

```diff
@@ -154,6 +154,29 @@ const ContractDetails = ({ systemIntake }: ContractDetailsProps) => {
   };
 
   const onSubmit = (values?: ContractDetailsForm) => {
+    const intakePayload = { ...values };
```

Comment:
could this error though if `values` is `null`/`undefined`?

Reply:
The desctructure evaluates to `{}` for `{...null}`